frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -275,5 +294,82 @@ function CollectionSectionHeading({
     </SidebarHeadingWrapper>
   );
 }
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export default MainNavbarView;
+
+function SidebarOnboardingSection({ initialState }: { initialState: boolean }) {
+  const applicationName = useSelector(getApplicationName);
+
+  return (
+    <Box
+      m={0}
+      bottom={0}
+      pos="fixed"
+      w={NAV_SIDEBAR_WIDTH}
+      bg="bg-white"
+      className={cx({ [CS.borderTop]: !initialState })}
+    >
+      <Box px="md" py="md">
+        {/*eslint-disable-next-line no-unconditional-metabase-links-render -- This link is only temporary. It will be replaced with an internal link to a page. */}
+        <ExternalLink href={getLearnUrl()} className={CS.noDecoration}>
+          {/* TODO: We currently don't have a `selected` state. Will be added in MS2 when we add the onboarding page. */}
+          <PaddedSidebarLink icon="learn">
+            {t`How to use ${applicationName}`}
+          </PaddedSidebarLink>
+        </ExternalLink>
+      </Box>
+      <Box px="xl" pb="md" className={cx({ [CS.borderTop]: initialState })}>
+        {initialState && (
+          <Text
+            fz="sm"
+            my="md"
+            style={{ lineHeight: "1.333" }}
+          >{t`Start by adding your data. Connect to a database or upload a CSV file.`}</Text>
+        )}
+
+        <Menu position="right-end" shadow="md">
+          <Menu.Target>
+            <Button
+              leftIcon={<Icon name="add_data" />}
+              fullWidth
+              // compact
```

Comment:
The design is using the compact version of the button but it doesn't match the `compact` styles that we currently have in the app.

I'm waiting for @romnitt to approve the normal button version first, after which I can remove this comment.